rotkehlchen/api/v1/schemas.py

```diff
@@ -2039,3 +2040,20 @@ class ReverseEnsSchema(AsyncIgnoreCacheQueryArgumentSchema):
 class SnapshotImportingSchema(Schema):
     balances_snapshot_file = FileField(allowed_extensions=['.csv'], required=True)
     location_data_snapshot_file = FileField(allowed_extensions=['.csv'], required=True)
+
+
+class AddressbookIdentifiersSchema(AsyncIgnoreCacheQueryArgumentSchema):
+    identifiers = fields.List(fields.String(required=True), load_default=None)
```

Comment:
With validated addresses you could check that this are addresses from either BTC/BCH/ETH/POLKADOT/AVAX